src/main/java/com/teamunify/i18n/I.java

```diff
@@ -618,8 +618,8 @@ else if (source.contains("."))
     }
 
     if (log.isDebugEnabled())
-      log.debug(String.format("Failed to parse date >%s< when using language settings for %s", source,
-                              s.locale.getLanguage()), e);
+      log.debug("Failed to parse date >{}< when using language settings for {}",
+			  source, s.locale.getLanguage(), e);
```

Comment:
Just for posterity:
https://stackoverflow.com/a/45054272/3485069

<details>
<summary>Excerpt</summary>

> As of SLF4J 1.6.0, in the presence of multiple parameters and if the last argument in a logging statement is an exception, then SLF4J will presume that the user wants the last argument to be treated as an exception and not a simple parameter.
>
> So, writing (in SLF4J version 1.7.x and later)
>
> ```java
> logger.error("one two three: {} {} {}", "a", "b", 
>         "c", new Exception("something went wrong"));
> ```
> Will do what you want to achieve...

</details>